topology/University of Chicago/UChicago/UChicagoCachingInfrastructure.yaml

```diff
@@ -74,3 +230,4 @@ Resources:
     AllowedVOs:
       - ANY_PUBLIC
       - LIGO
+
```

Comment:
```suggestion
```

Reply:
Watch out for rogue whitespace in your commits! https://osg-htc.org/technology/software/git-software-development/#avoid-whitespace-noise